### PR TITLE
Transition from 'artifact tree' to 'artifact dependency graph (ADG)' in spec

### PR DESCRIPTION
**Pull request by edwarnicke**
Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -30,20 +30,20 @@ Type text.
 
 ##	Scope (mandatory)
 
-Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
 data structures for artifacts.  Including but not limited to:
```
Comment (jeff-schutt) [inline SPEC.md:34]: Remove EOL between "related" and "data"?

File: SPEC.md [hunk 2]
```diff
@@ -30,20 +30,20 @@ Type text.
...
 
 - formats for artifact identifiers
 - formats for specifying graph relationships between artifacts
-- manner of embedding identifiers for artifact trees, directed acyclic graphs (DAGs), and other related 
+- manner of embedding identifiers for ADGs, and other related
 data structures in artifacts of various types
```
Comment (jeff-schutt) [inline SPEC.md:39]: Remove EOL between "related" and "data"?

File: SPEC.md [hunk 3]
```diff
@@ -30,20 +30,20 @@ Type text.
...
-  - conveying artifact trees, DAGs, and other related data structures
-  - embedding identifiers for artifact trees, DAGs, and other related data structures ids in artifacts
-  - manners of conveyance of artifact trees, DAGs, and other related data structures
-  - descriptions of use cases for which artifact trees, directed acyclic graphs, and other related data structures may be used
-  - guidance on metadata which references artifact trees, DAGs, and other related data structures
+  - constructing ADGss, and other related data structures
+  - conveying ADGs, and other related data structures
+  - embedding identifiers for ADGs, and other related data structures ids in artifacts
+  - manners of conveyance of ADGs, and other related data structures
+  - descriptions of use cases for which directed acyclic graphs, and other related data structures may be used
```
Comment (jeff-schutt) [inline SPEC.md:46]: Swap "directed acyclic graphs" for ADG as ADGs are defined as DAGs?

File: SPEC.md [hunk 4]
```diff
@@ -30,20 +30,20 @@ Type text.
...
 data structures in artifacts of various types
+- guidance on metadata which references ADGs, and other related data structures
 - guidance for build tools for:
-  - constructing artifact artifact trees, DAGs, and other related data structures
-  - conveying artifact trees, DAGs, and other related data structures
-  - embedding identifiers for artifact trees, DAGs, and other related data structures ids in artifacts
-  - manners of conveyance of artifact trees, DAGs, and other related data structures
-  - descriptions of use cases for which artifact trees, directed acyclic graphs, and other related data structures may be used
-  - guidance on metadata which references artifact trees, DAGs, and other related data structures
+  - constructing ADGss, and other related data structures
```
Comment (jeff-schutt) [inline SPEC.md:42]:
```suggestion
  - constructing ADGs, and other related data structures
```

File: SPEC.md [hunk 5]
```diff
@@ -151,7 +151,7 @@ is the git object id of the GitBOM document.
 
 ### Build tool persistence of related metadata
 
-When persisting an output artifact to a file system, if the build tool has additional metadata to persist that makes refernce to the artifact tree,
+When persisting an output artifact to a file system, if the build tool has additional metadata to persist that makes refernce to the Artifact Dependency Graph (ADG),
```
Comment (jeff-schutt) [inline SPEC.md:154]:
```suggestion
When persisting an output artifact to a file system, if the build tool has additional metadata to persist that makes reference to the Artifact Dependency Graph (ADG),
```

File: SPEC.md [hunk 6]
```diff
@@ -30,20 +30,20 @@ Type text.
 
 ##	Scope (mandatory)
 
-Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
```
Comment (jeff-schutt) [inline SPEC.md:33]:
```suggestion
Specifies procedures for constructing and conveying Artifact Dependency Graphs (ADGs), and other related
```

**Review by jeff-schutt — CHANGES_REQUESTED** [review]

File: SPEC.md (hunk 1 above)
Comment (edwarnicke) [reply SPEC.md:34]: This was semi intentional.

When rendered as markdown, the EOL will be disappear: markdown considers it a single paragraph.
When editing in an editor... it makes the line a reasonable length.

File: SPEC.md (hunk 5 above)
Comment (edwarnicke) [reply SPEC.md:154]: Fixed in latest commit.

File: SPEC.md (hunk 3 above)
Comment (edwarnicke) [reply SPEC.md:46]: Fixed in latest commit.

File: SPEC.md (hunk 4 above)
Comment (edwarnicke) [reply SPEC.md:42]: Fixed in latest commit.

File: SPEC.md (hunk 6 above)
Comment (edwarnicke) [reply SPEC.md:33]: Fixed in latest commit.

**Review by jeff-schutt — APPROVED** [review]
LGTM!

**Pull request merged**